docker.Makefile

```diff
@@ -8,6 +8,7 @@ DEV_DOCKER_IMAGE_NAME = docker-cli-dev
 LINTER_IMAGE_NAME = docker-cli-lint
 CROSS_IMAGE_NAME = docker-cli-cross
 MOUNTS = -v `pwd`:/go/src/github.com/docker/cli
+VERSION = $(shell cat VERSION)
```

Comment:
actually do you mind updating `scripts/build/.variables` to use `$(cat VERSION)` as the default instead of `unknown-version`?